ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/FilterChips.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { Globe } from 'react-feather';
+import { Flex, FlexItem, ChipGroup, Chip } from '@patternfly/react-core';
+
+import { SearchFilter } from 'types/search';
+import { VulnerabilitySeverityLabel, FixableStatus, DefaultFilters } from './types';
+
+import './FilterChips.css';
+
+type FilterChipProps = {
+    isGlobal?: boolean;
+    name: string;
+};
+
+function FilterChip({ isGlobal, name }: FilterChipProps) {
+    if (isGlobal) {
+        return (
+            <Flex alignItems={{ default: 'alignItemsCenter' }} flexWrap={{ default: 'nowrap' }}>
+                <Globe height="15px" />
+                {name}
+            </Flex>
+        );
+    }
+    return <Flex>{name}</Flex>;
+}
+
+type FilterChipsProps = {
+    defaultFilters: DefaultFilters;
+    searchFilter: SearchFilter;
+    onDeleteGroup: (category) => void;
+    onDelete: (category, chip) => void;
+};
+
+function FilterChips({ defaultFilters, searchFilter, onDeleteGroup, onDelete }: FilterChipsProps) {
+    const deployments = (searchFilter.DEPLOYMENT as string[]) || [];
+    const cves = (searchFilter.CVE as string[]) || [];
+    const images = (searchFilter.IMAGE as string[]) || [];
+    const namespaces = (searchFilter.NAMESPACE as string[]) || [];
+    const clusters = (searchFilter.CLUSTER as string[]) || [];
+    const severities = (searchFilter.Severity as VulnerabilitySeverityLabel[]) || [];
+    const fixables = (searchFilter.Fixable as FixableStatus[]) || [];
+
+    return (
+        <Flex spaceItems={{ default: 'spaceItemsXs' }}>
+            <FlexItem className="pf-u-pt-xs">
+                <ChipGroup
```

Comment:
Small nit: each search category option here will leave an empty div with a 4px margin due to `pf-u-pt-xs` in the chip section that gets pretty noticeable if there are a lot of empty categories. Not worth keeping this PR from being merged though, we can get it during a later round of visual tweaks.
<img width="784" alt="image" src="https://user-images.githubusercontent.com/1292638/227016448-e9be2314-dcd4-43ee-8b90-4bb927737423.png">